.github/workflows/on-demand-vrfv2plus-load-test.yml

```diff
@@ -0,0 +1,112 @@
+name: On Demand VRFV2 Plus Load Test
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: Network to run tests on
+        type: choice
+        options:
+          - "SIMULATED"
+          - "GOERLI"
+          - "OPTIMISM_GOERLI"
+          - "ARBITRUM_GOERLI"
+          - "CELO_ALFAJORES"
+          - "CELO_MAINNET"
+          - "BASE_GOERLI"
+          - "BASE_MAINNET"
+          - "BSC_MAINNET"
+          - "BSC_TESTNET"
+          - "SCROLL_SEPOLIA"
+          - "SCROLL_MAINNET"
+      fundingPrivateKey:
+        description: Private funding key (Skip for Simulated)
+        required: false
+        type: string
```

Comment:
Just curious but how is this transmitted to the GHA instance that will be running the test?

Reply:
So [here](https://github.com/smartcontractkit/chainlink/blob/ae39b72435349d5a444ff2681725f9646816fc9c/.github/workflows/on-demand-vrfv2plus-load-test.yml#L93) it is set as a ENV variable to the test